packages/server/test/util/factories.ts

```diff
@@ -110,3 +111,9 @@ export const InteractionFactory = new Factory(InteractionModel)
   .assocOne('course', CourseFactory)
   .assocOne('user', UserFactory)
   .attr('timestamp', new Date());
+
+export const OrganizationCourseFactory = new Factory(OrganizationCourseModel)
+  .attr('organizationId', 1)
```

Comment:
I am unsure if passing constant values into the factory is a great idea? What about collisions in the database with FKs? i.e. what if we want to create multiple factories of this constant?